pkgs/top-level/python-aliases.nix

```
@@ -1,4 +1,4 @@
-lib: self: super:
+{ lib, fetchPypi }: self: super:
```

```suggestion
{ lib, pkgs }: self: super:
```
if we add any other packages here in the future

---

Honestly I am not sure and if it breaks splicing. I don't know if we have a predecessor for this. Maybe it is enough to use a throw like the other packages we move to pkgs.

---

The package set should already be part of `super`, so `super.pkgs.fetchPypi`.

---

good point, updated the PR